data/MacOSX/Getting-Started.txt

```diff
@@ -1,9 +1,7 @@
 GETTING (THE SSR) STARTED
 
-You should have a proper version of Jack installed by now (32 or 64 bit - matching the architecture of the SSR build you are using). If not, get it from http://www.jackosx.com/
+You should have a proper version of Jack installed by now (32 or 64 bit - matching the architecture of the SSR build you are using). If not, get it from https://jackaudio.org/ or from Homebrew. You can also obtain QJackCtl from there. 
```

Comment:
It would probably be better to move the content of this whole file to the manual and add a link to it here?

Reply:
This content is actually in the manual. It might be worth keeping this here so that macOS bundle users are provided with this information directly.

Reply:
You are right, I didn't consider that this is included in the DMG.